Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -454,6 +456,8 @@ for this property.
 
 - (void)_textFragmentDirectiveFromSelectionWithCompletionHandler:(void(^)(NSURL *))completionHandler WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
 
+- (void)_textFragmentRangesWithCompletionHandler:(void(^)(NSArray<NSValue *> * fragmentRanges))completionHandler WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
```

Comment:
If it's really only for testing, I'd recommend putting it in `WKWebViewPrivateForTesting.h` (optionally, with a `-ForTesting` suffix in the name).